index.bs

```diff
@@ -6600,8 +6600,8 @@ Note: If PRF results are obtained during [=registration=] then the [=[RP]=] MUST
 
 : Client extension processing ([=authentication extension|authentication=])
 ::
-      1. If {{PublicKeyCredentialRequestOptions/userVerification}} has the value {{UserVerificationRequirement/preferred}}, return a {{DOMException}} whose name is “{{NotSupportedError}}”.
       1. If {{AuthenticationExtensionsPRFInputs/evalByCredential}} is not empty but {{PublicKeyCredentialRequestOptions/allowCredentials}} is empty, return a {{DOMException}} whose name is “{{NotSupportedError}}”.
+      1. If any [=map/key=] in {{AuthenticationExtensionsPRFInputs/evalByCredential}} is the empty string, or is not a valid [=base64url encoding=], return a {{DOMException}} whose name is “{{NotSupportedError}}”.
```

Comment:
Is `NotSupportedError` the most suitable error name here? I'm thinking something more like [`SyntaxError`](https://webidl.spec.whatwg.org/#syntaxerror) or [`DataError`](https://webidl.spec.whatwg.org/#dataerror) might be a better fit (at least for the second line, probably both lines?). The Web IDL spec even includes an example that sounds a lot like what we're doing here:

>"[SyntaxError](https://webidl.spec.whatwg.org/#syntaxerror)" [DOMException](https://webidl.spec.whatwg.org/#idl-DOMException) is used to report parsing errors in web APIs, for example when parsing selectors

...although I suppose that's probably referring to CSS selectors, so maybe not quite as close a match.

Reply:
Thanks. I think you're right that SyntaxError makes more sense here. Done.